src/bundle/Resources/public/js/scripts/core/date.time.picker.js

```diff
@@ -120,10 +120,12 @@ class DateTimePicker {
                 return;
             }
 
-            if (isMinute) {
-                flatpickrDate.setMinutes(value);
-            } else {
+            if (isHour) {
                 flatpickrDate.setHours(value);
+            } else if (isMinute) {
+                flatpickrDate.setMinutes(value);
+            } else if (isSecond) {
+                flatpickrDate.setSeconds(value);
```

Comment:
and with changing to single variable this can be remade
```suggestion
            switch (type) {
                case 'hours':
                    flatpickrDate.setHours(value);
                    break;
                case 'minutes':
                    flatpickrDate.setMinutes(value);
                    break;
                case 'seconds':
                    flatpickrDate.setSeconds(value);
                    break;
            }
```
or ifs if someone doesn't like switches ;)